From 1e9986912573b998347dcc918e03cae51e0d1e6d Mon Sep 17 00:00:00 2001
From: Maya Matuszczyk <maccraft123mc@gmail.com>
Date: Thu, 14 Nov 2024 18:14:36 +0100
Subject: [PATCH] arm64: dts: mt8183-kukui: Disable dpi

!!! CADMIUM HACK !!!

This works around an issue where dpi component tries to get a drm bridge,
fails and then the rest of the driver falls over, because dpi driver
returned -ENODEV

mediatek-dpi 14015000.dpi: error -ENODEV: Failed to get bridge

Signed-off-by: Maya Matuszczyk <maccraft123mc@gmail.com>
---
 arch/arm64/boot/dts/mediatek/mt8183-kukui.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/mediatek/mt8183-kukui.dtsi b/arch/arm64/boot/dts/mediatek/mt8183-kukui.dtsi
index 22924f61ec9e..a3822185c3c4 100644
--- a/arch/arm64/boot/dts/mediatek/mt8183-kukui.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt8183-kukui.dtsi
@@ -259,6 +259,10 @@ &cpu7 {
 	proc-supply = <&mt6358_vproc11_reg>;
 };
 
+&dpi0 {
+	status = "disabled";
+};
+
 &dsi0 {
 	status = "okay";
 	#address-cells = <1>;
-- 
2.47.0

